app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,12 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.DayNight.DarkActionBar">
         <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:background">@color/colorBackground</item>
```

Comment:
You could just omit this line, that way you get sensible default background colors: purple and white for the AppBar/rest in light mode, and almost-black gray in dark mode. I'll fix that in a follow-up commit.

Reply:
The almost-black gray is better than blinding white for sure but I'd quite like a pure black option as well for dark mode. What's the best way to add that? Maybe a checkbox in the options menu?

Reply:
I don't have a strong opinion on this, so if you find a way to have completely black background, but keep the light case unchanged, I would accept that. An option for this is probably an overkill :)